vocabularies/trs.ttl

```diff
@@ -0,0 +1,178 @@
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Iranian_calendars> ;
+  skos:definition """The Iranian calendars or Iranian chronology (Persian: گاه‌شماری ایرانی‎, Gāh-Shomāri-ye Irāni) are a succession of calendars invented or used for over two millennia in Iran also known as Persia. One of the longest chronological records in human history, the Iranian calendar has been modified time and time again during its history to suit administrative, climatic, and religious purposes.
+
+As one of the few calendars designed in the era of accurate positional astronomy, the Iranian calendar uses a very complex leap year structure which makes it the most accurate solar calendar in use today.""" ;
+  skos:prefLabel "Iraian Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:IslamicCalendar
+  a skos:Concept ;
+  a time:TRS ;
+  dct:source <https://aa.usno.navy.mil/faq/docs/calendars.php> ;
+  rdfs:seeAlso <https://aa.usno.navy.mil/faq/docs/islamic.php> ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Islamic_calendar> ;
+  skos:definition "The Islamic, Muslim, or Hijri calendar (Arabic: التقويم الهجري‎ at-taqwīm al-hijrī) is a lunar calendar consisting of 12 lunar months in a year of 354 or 355 days. It is used to determine the proper days of Islamic holidays and rituals, such as the annual period of fasting and the proper time for the pilgrimage to Mecca." ;
+  skos:prefLabel "Islamic Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:JulianCalendar
+  a skos:Concept ;
+  a time:TRS ;
+  dct:source <https://aa.usno.navy.mil/faq/docs/calendars.php> ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Julian_calendar> ;
+  skos:definition "The Julian calendar, proposed by Julius Caesar in 708 AUC (46 BC/BCE), was a reform of the Roman calendar. It took effect on 1 January 709 AUC (45 BC/BCE), by edict. It was designed with the aid of Greek mathematicians and Greek astronomers such as Sosigenes of Alexandria. " ;
+  skos:prefLabel "Julian Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:JulianDay
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Julian_day> ;
+  skos:definition """Julian day is the continuous count of days since the beginning of the Julian Period and is used primarily by astronomers, and in software for easily calculating elapsed days between two events (e.g. food production date and sell by date).
+
+The Julian Day Number (JDN) is the integer assigned to a whole solar day in the Julian day count starting from noon Universal time, with Julian day number 0 assigned to the day starting at noon on Monday, January 1, 4713 BC, proleptic Julian calendar (November 24, 4714 BC, in the proleptic Gregorian calendar), a date at which three multi-year cycles started (which are: Indiction, Solar, and Lunar cycles) and which preceded any dates in recorded history. For example, the Julian day number for the day starting at 12:00 UT on January 1, 2000, was 2 451 545. """ ;
```

Comment:
Is this still part of the definition? Should be on a single line.
Also ends with many quotation marks.

Reply:
It is part of the definition but the definition's enclosed in triple quotes """ which, in Turtle syntax means line breaks are respected. See the Turtle spec's section on [quoted literals](https://www.w3.org/TR/turtle/#turtle-literals).